### PR TITLE
project-chatbot Mary Zaia

### PR DESCRIPTION
**Pull request by MaryZaia**
link to the site: https://thecoffeeshopbot.netlify.app

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -1,51 +1,151 @@
...
 
 // Starts here
-const greetUser = () => {
+
+//Greeting
+const greeting = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage("Hello, What's your name?", "bot");
```
Comment (NinaWald) [inline code/script.js:39]: Very nice, good job!

File: code/script.js [hunk 2]
```diff
@@ -1,51 +1,151 @@
...
+  inputWrapper.innerHTML = `
+  <button id="small" type="submit" class="chat-btn">Small</button>
+  <button id="large" type="submit" class="chat-btn">Large</button>
+  `
+  document.getElementById("small").addEventListener("click", smallSize);
+  document.getElementById("large").addEventListener("click", largeSize);
+};
+
+ 
+// size choice
```
Comment (NinaWald) [inline code/script.js:102]: Very easy to to see what happens in the code when you write comments

File: code/script.js [hunk 3]
```diff
@@ -1,51 +1,151 @@
...
+
+  document.getElementById("takeAwayBtn").addEventListener("click", () => whereToDrink("take away"))
+  document.getElementById("drinkHereBtn").addEventListener("click", () => whereToDrink("drink here"))
+  ;}
+
+// Where to drink, take away or sit here
+// pass  largeChoice, smallChoice, coffeeChoice, teaChoice as props for summary?? 
+const whereToDrink = (whereToDrink) => {
+
+  if (whereToDrink === "take away"){
```
Comment (NinaWald) [inline code/script.js:135]: Is span where the buttons disappear?

**Review by NinaWald — COMMENTED** [review]
Good job !

File: README.md [hunk 4]
```diff
@@ -1,13 +1,3 @@
-# Project Name
+# Project chatbot
 
-Replace this readme with your own information about your project.
```
Comment (MaryZaia) [inline README.md:3]: test

File: README.md [hunk 5]
```diff
@@ -1,13 +1,3 @@
...
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
-
-## View it live
-
```
Comment (MaryZaia) [inline README.md:12]: test1

File: code/index.html [hunk 6]
```diff
@@ -12,16 +12,16 @@
...
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
+    <h1>Welcome to the coffee shop</h1>
+    <h2>Order here</h2>
     <main>
       <section class="chat" id="chat"></section>
 
       <div class="input-wrapper" id="input-wrapper">
         <form id="name-form">
           <label for="name-input">Name</label>
```
Comment (MaryZaia) [inline code/index.html:22]: test3

**Review by MaryZaia — COMMENTED** [review]
test4

**Pull request closed without merging**